examples/functions.jsonnet

```diff
@@ -18,6 +18,13 @@ limitations under the License.
 // Default arguments are like Python:
 local my_function(x, y=10) = x + y;
 
+// Define a local multiline function. Parenthesis are optionally used as
+// a block delimiter - e.g. curly braces in C-like languages.
+local multiline_function(x) = (
```

Comment:
You don't need the parens and I think as long as there is indentation, there's not much point adding them.  You can just put the ; immediately after the temp.

Reply:
Happy to remove these. I imagine the parser isn't whitespace sensitive and knows to terminate the `multiline_function` definition when it encounters the terminal `temp`. What is the most precise word that encompasses the terminal statements here: e.g.  it could be either `temp` or `std.sort(...)` or `{"foo": "bar"}`.

Reply:
It's the `;` that ends it, so maybe:
(I'm keeping this very short so it looks good on the website)

```
// Define a local multi-line function.
local multiline_function(x) =
  local temp = x * 2;  // One can nest locals.
  temp;  // Every local ends with a semicolon.
```

Reply:
It'd make more sense if you actually did something with temp, even if it's just `[temp, temp + 1]` or something like that.

Reply:
You can paste it into the tutorial to make sure it actually fits

Reply:
Updated according to these suggestions.